TestModels/SimpleTypes/SimpleDouble/runtimes/rust/src/operation/get_double/_get_double_input.rs

```diff
@@ -0,0 +1,60 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+#[allow(missing_docs)] // documentation missing in model
+#[non_exhaustive]
+#[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
+pub struct GetDoubleInput {
+    #[allow(missing_docs)] // documentation missing in model
+    pub value: ::std::option::Option<::std::vec::Vec<::std::primitive::u8>>,
```

Comment:
Ah sorry I wasn't clearer, this should still be `f64`, cause that's the idiomatic Rust type for a Smithy long.

But since the Dafny-generated type is `dafny_runtime::Sequence<u8>`, the conversion `from_dafny` function needs to call something like `f64.from_be_bytes(...)` on the sequence of bytes.